app/application/use_cases/reset_vault.py

```
@@ -0,0 +1,97 @@
+        async def _send_reset_if_online() -> None:
+            online_result = self._ws_manager.is_vault_online(vault.id)
+            online = await online_result if inspect.isawaitable(online_result) else online_result
+            if not online:
+                return
+
+            send_result = self._ws_manager.send_to_vault(vault.id, {"command": "reset"})
+            if inspect.isawaitable(send_result):
+                await send_result
+
+        try:
+            asyncio.create_task(_send_reset_if_online())
```

Ruff (0.15.2)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)